src/test/java/website/frontrow/util/PointTest.java

```diff
@@ -161,4 +173,24 @@ public void lengthTest2()
         Point a = new Point(0, 2);
         assertEquals(2, a.length(), 0);
     }
+
+    /**
+     * Test angle.
+     */
+    @Test
+    public void angleTest1()
+    {
+        Point a = new Point(1, 0);
```

Comment:
Not an abbreviation, just had no clue what to name the other point.
Because pretty much any name works.

Any suggestions?